app/src/main/java/org/oppia/android/app/topic/lessons/ChapterSummaryViewModel.kt

```diff
@@ -23,14 +24,22 @@ class ChapterSummaryViewModel(
   }
 
   fun computeChapterPlayStateIconContentDescription(): String {
-    return if (chapterPlayState == ChapterPlayState.COMPLETED) {
-      resourceHandler.getStringInLocaleWithWrapping(
-        R.string.chapter_completed, (index + 1).toString(), chapterTitle
-      )
-    } else {
-      resourceHandler.getStringInLocaleWithWrapping(
-        R.string.chapter_in_progress, (index + 1).toString(), chapterTitle
-      )
+    return when (chapterPlayState) {
+      ChapterPlayState.COMPLETED -> {
+        resourceHandler.getStringInLocaleWithWrapping(
+          R.string.chapter_completed, (index + 1).toString(), chapterTitle
+        )
+      }
+      ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES -> {
+        resourceHandler.getStringInLocaleWithWrapping(
+          R.string.chapter_prerequisite_title_label, (index).toString(), previousChapterTitle
```

Comment:
```suggestion
          R.string.chapter_prerequisite_title_label, index.toString(), previousChapterTitle
```

Reply:
This has been modified with the latest changes.